classes/Task.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private int $customer_id;
+    private ?int $executor_id;
+    private string $current_status = self::STATUS_NEW;
+
+    const STATUS_NEW = 'status_new';
+    const STATUS_CANCELED = 'status_canceled';
+    const STATUS_IN_WORK = 'status_in_work';
+    const STATUS_PERFORMED = 'status_performed';
+    const STATUS_FAILED = 'status_failed';
+    const ACTION_CANCEL = 'action_cancel';
+    const ACTION_RESPOND = 'action_respond';
+    const ACTION_EXECUTE = 'action_execute';
+    const ACTION_REFUSE = 'action_refuse';
+
+    public function __construct(int $customer_id, int $executor_id = null)
+    {
+        $this->executor_id = $executor_id;
+        $this->customer_id = $customer_id;
+    }
+
+    // Объект класса создаёт заказчик, исполнитель, скорее всего, подключится позже
+    // Функция установки id исполнителя задания
+    public function setExecutorId(int $executor_id)
+    {
+        $this->executor_id = $executor_id;
+    }
+
+    public function getCurrentStatus(): string
+    {
+        return $this->current_status;
+    }
+
+    public function getStatusMap(): array
```

Comment:
В методах, которые возвращают массив или которые принимают массивы в качестве аргументов, лучше сразу указывать типы элементов массива в виде аннотаций
```suggestion

    /**
     * @return array<string>
     */
    public function getStatusMap(): array
```
Для указания типа аргумента используют аннотацию `@param`
```php
/**
 * @param array<int> $someArgument
 */
private function someFunction(array $someArgument): void
```